plugins/gatsby-transformer-garden-svg/tokens.json

```diff
@@ -0,0 +1,232 @@
+[
+  {
+    "name": "Add New",
+    "icon": "plus-stroke",
+    "alternatives": []
+  },
+  {
+    "name": "Alert Error",
+    "icon": "alert-error-stroke",
+    "alternatives": ["circle", "exclamation point"]
+  },
+  {
+    "name": "Alert Info",
+    "icon": "info-stroke",
+    "alternatives": ["information", "i", "circle"]
+  },
+  {
+    "name": "Alert Success",
+    "icon": "check-circle-stroke",
+    "alternatives": ["checkmark"]
+  },
+  {
+    "name": "Alert Warning",
+    "icon": "alert-warning-stroke",
+    "alternatives": ["danger", "triangle", "exclamation point"]
+  },
+  {
+    "name": "Attachment",
+    "icon": "paperclip-stroke",
+    "alternatives": ["file"]
+  },
+  {
+    "name": "Avatar",
+    "icon": "user-solo-stroke",
+    "alternatives": ["profile", "person", "user"]
```

Comment:
Is "profile", "person", and "user" supposed to be tokens? If so, would it make more sense to name this value "tokens" instead of "alternatives"?

Reply:
Good question, I'm not sure if they are alternate terms or tokens. @exelarios @bethanybeech any idea on this?

Reply:
JZ suggested the key name for alternate terms to be `synonyms`. Any thoughts on that too?

Reply:
It's in the proposed garden icon tokens google sheet, those "alternatives" words come from the "metadata" column. "Profile", "person", "user" are words Bethany and I came up with to account for other search keywords that a user would think of to put in to search for the icon.

In my opinion, a more accurate single word name for them is "keywords" or more specifically "search-keywords"

Reply:
What is a "token"? And which line represents the token identifier?

I guess "Avatar" is the "token" here? If so then yeah "alternatives" or "synonyms" seem like a good key for `["profile", "person", "user"]`. But then shouldn't the key "name" be "token" instead since that _is_ the token?

Reply:
Big plus one to Henry. Having more literal and clear labels will help reduce the confusion like name --> token

Reply:
Thanks for the clarification. Yeah `synonyms` or `keywords` seem good 👍🏽 .

Reply:
Thanks for clearing that up. I feel like we had different definitions of what `token` actually was.